src/domains/citizen-auth-app/04_function_lollipop.tf

```
@@ -247,136 +179,459 @@ module "function_lollipop_staging_slot_itn" {
 }
 
 resource "azurerm_monitor_autoscale_setting" "function_lollipop_itn" {
-  name                = format("%s-autoscale-01", module.function_lollipop_itn.name)
+  name                = "${replace(module.function_lollipop_itn.name, "fn", "as")}-01"
```

More consistent and less error prone.
```suggestion
  name                = "${replace(module.function_lollipop_itn.name, "-fn-", "-as-")}-01"
```